engine/liveupdate/src/liveupdate.cpp

```diff
@@ -946,7 +946,10 @@ namespace dmLiveUpdate
 
         dmLogInfo("Liveupdate folder located at: %s", g_LiveUpdate.m_AppSupportPath);
 
-        g_LiveUpdate.m_JobThread = dmJobThread::Create("liveupdate_jobs");
+        dmJobThread::JobThreadCreationParams job_thread_create_param;
+        job_thread_create_param.m_ThreadName = "liveupdate_jobs";
+
+        g_LiveUpdate.m_JobThread = dmJobThread::Create(1, &job_thread_create_param);
```

Comment:
Liveupdate has its own thread right now, but we could potentially use the main engine workers for this as well and process more jobs in parallell. It felt like a bigger change right now since I don't know this system very well